pkg/autoscaler/statforwarder/processor.go

```diff
@@ -62,3 +89,17 @@ func (p *bucketProcessor) shutdown() {
 		p.conn.Shutdown()
 	}
 }
+
+// TODO(yanweiguo): use websocket.NewDurableSendingConnectionGuaranteed instead once knative.dev/pkg
```

Comment:
For purpose of your work, I'd suggest you override go.mod pointing to your head pkg, which you can revert on tuesday. Then you won't have to do the double work.

Reply:
Since this PR won't change anything as the forwarder is not used anywhere, I'd like to get it in ASAP. I'd like to pin to head pkg in https://github.com/knative/serving/pull/9465 where we enable HA for autoscaler.

Reply:
Done